app/reducers/quotes.ts

```diff
@@ -13,8 +13,8 @@ export type QuoteEntity = {
   source: string;
   approved: boolean;
   contentTarget: string;
-  reactionsGrouped: Array<ReactionEntity>;
-  reactions: Array<ReactionEntity>;
+  reactionsGrouped: ReactionsGrouped[];
+  reactions: unknown;
```

Comment:
Why is this `unknown`?

Reply:
Good question! That's because I have no idea what it is. I'm actually pretty sure it never is anything at all.

Reply:
I'll just remove it, pretty sure it isn't used for anything